cli/compose/types/types.go

```diff
@@ -305,6 +305,7 @@ type IPAMPool struct {
 
 // VolumeConfig for a volume
 type VolumeConfig struct {
+	Name       string
```

Comment:
@dnephin should `External.Name` below (https://github.com/docker/cli/pull/306/files#diff-80e4727ffc215ce8081027b2780b1494L317) be marked _deprecated_ ?